charts/spire/charts/spire-server/values.yaml

```diff
@@ -34,6 +34,8 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+jobAnnotations: {}
```

Comment:
From a practical point of view. 

- When will we use this?
- Do we need to be able to annotate differently per job?
- If so do we need post-upgrade-job annotations and pre-delete job annotations etc?

This is an open question, no thought or ideas on that myself. Trying to ensure we aren't adding things that need changes soon again, so we can keep the API stable.